src/Tables.jl

```diff
@@ -123,6 +123,7 @@ rowaccess(::Type{T}) where {T} = false
 columnaccess(x::T) where {T} = columnaccess(T)
 columnaccess(::Type{T}) where {T} = false
 schema(x) = nothing
+materializer(x) = identity
```

Comment:
How useful is this default? This means we'll return a row- or a column-table depending on the internal implementation. Wouldn't it be better to throw an error, or always use the same kind of representation?

Reply:
My thinking is that it wouldn't break anything, and then the user can decide what to do with the row or column table that's returned, rather than possibly making a bunch of copies to go from row to column or vice versa.  I'd be okay with throwing an error or choosing `rowtable`/`columntable`